src/DeleteOldSoftDeletes.php

```diff
@@ -48,7 +62,7 @@ private function deleteOldSoftDeletesForModel($modelName, $modelConfig, $daysBef
     {
         $daysBeforeDeletion = empty($modelConfig['days']) ? $daysBeforeDeletion : $modelConfig['days'];
 
-        $affectedRows = DB::table((new $modelName)->getTable())
+        $affectedRows = $this->db->table((new $modelName)->getTable())
```

Comment:
It works in the package, but dies when I run it in Karani.

`BadMethodCallException: Call to undefined method Illuminate\Database\Query\Builder::withTrashed()`